components/tyk-dashboard/values.yaml

```diff
@@ -0,0 +1,151 @@
+# Determines whether or not the dashboard component should be installed.
+enabled: true
+# Dashboard will only bootstrap if the master bootstrap option is set to true.
+bootstrap: true
+
+
+# The hostname to bind the Dashboard to.
+hostName: tyk-dashboard.local
+# If set to true the Dashboard will use SSL connection.
+# You will also need to set the:
+# - TYK_DB_SERVEROPTIONS_CERTIFICATE_CERTFILE
+# - TYK_DB_SERVEROPTIONS_CERTIFICATE_KEYFILE
+# variables in extraEnvs object array to define your SSL cert and key files.
+tls: false
+# Dashboard license.
+license: ""
+# Dashboard admin information.
+adminUser:
+  firstName: admin
+  lastName: user
+  email: default@example.com
+  # Set a password or a random one will be assigned.
+  password: "123456"
+# Dashboard Organisation information.
+org:
+  name: Default Org
+  # The hostname to bind the Portal to.
+  cname: tyk-portal.local
+
+replicaCount: 1
+containerPort: 3000
+image:
+  repository: tykio/tyk-dashboard
+  tag: v5.0.0
+  pullPolicy: Always
+service:
+  type: NodePort
+  port: 3000
+  externalTrafficPolicy: Local
+  annotations: {}
+
+#  - secretName: chart-example-tls
+#    hosts:
+#      - chart-example.local
+resources: {}
+  # We usually recommend not to specify default resources and to leave this
+  # as a conscious choice for the user. This also increases chances charts
+  # run on environments with little resources, such as Minikube. If you do
+  # want to specify resources, uncomment the following lines, adjust them
+  # as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #  cpu: 100m
+  #  memory: 128Mi
+  # requests:
+  #  cpu: 100m
+#  memory: 128Mi
+securityContext:
+  runAsUser: 1000
+  fsGroup: 2000
+nodeSelector: {}
+tolerations: []
+affinity: {}
+extraEnvs: []
+## extraVolumes A list of volumes to be added to the pod
+## extraVolumes:
+##   - name: ca-certs
+##     secret:
+##       defaultMode: 420
+##       secretName: ca-certs
+extraVolumes: []
+## extraVolumeMounts A list of volume mounts to be added to the pod
+## extraVolumeMounts:
+##   - name: ca-certs
+##     mountPath: /etc/ssl/certs/ca-certs.crt
+##     readOnly: true
+extraVolumeMounts: []
+mounts: []
+
+secrets:
+  # tyk_analytics.conf tyk_api_config.secret
+  # tyk_analytics.conf shared_node_secret
+  # tyk.conf node_secret
+  # tyk.conf secret
+  APISecret: CHANGEME
+  # tyk_analytics.conf admin_secret
+  # tib.conf TykAPISettings.GatewayConfig.AdminSecret
+  # tib.conf TykAPISettings.DashboardConfig.AdminSecret
+  AdminSecret: "12345"
+  # If you don't want to store plaintext secrets in the Helm value file and would
+  # rather provide the k8s Secret externally please populate the value below
+  useSecretName: ""
+
+enableIstioIngress: false
+
+gateway:
+  hostName: "hname"
+  port: 8080
+  host: "host"
+
+backend: mongo
+
+postgres:
+  host: tyk-postgres-postgresql.tyk.svc.cluster.local
+  port: 5432
+  user: postgres
+  password:
+  database: tyk_analytics
+  sslmode:
+
+
+redis:
```

Comment:
i believe it should be located under `global` field as we did in other components. what do you think?